src/main/scala/com/actian/spark_vector/colbuffer/real/DoubleColumnBuffer.scala

```diff
@@ -26,8 +26,18 @@ private class DoubleColumnBuffer(p: ColumnBufferBuildParams) extends ColumnBuffe
   override def get(buffer: ByteBuffer): Double = buffer.getDouble()
 }
 
-/** Builds a `ColumnBuffer` object for `float`, `float8`, `double precision` types. */
+private class MoneyColumnBuffer(p: ColumnBufferBuildParams) extends DoubleColumnBuffer(p) {
```

Comment:
yes indeed, tested only the unload (from an existent table) previously. 

the load shows that it should take the 1e2 in account